src/resources/views/vendor/backpack/crud/fields/select_page_template.blade.php

```diff
@@ -66,12 +66,15 @@ function strip_last_template_parameter(url) {
             }
 
             jQuery(document).ready(function($) {
+                $('#select_template').data('current', $('#select_template').val());
```

Comment:
```suggestion
                let currentTemplate = $('#select_template').val();
```

Reply:
I guess I agree with the premise (that we don't use it multiple times per form), but not with the conclusion (that we should do something about that) 😂 

I think:
- we have no idea how people are using the field - they might as well be using it multiple times per page - who knows; 
- all other fields work multiple times per page, so ideally this should too (if not now then in the future);
- changing the source of truth could also break other custom integrations, that use _that_ as a source of truth;

That makes me think it's better to just leave it as-is.

Reply:
(in case this data-attribute was there before this PR)

Reply:
@tabacitu I think it's ok to merge this as is 😅
When I said people don't use this multiple times per page, it's because in the field we use IDs to identify it, so it will throw JS errors when using it twice.

The way it's coded is correct, it could be just a little bit improved by using variables just because the field doesn't have to care about multiple instances.

Reply:
@promatik @tabacitu - Yes I was using ID as selector because the field does not have any `bpFieldInitxxxxElement` method attached to it 😅 

I'm also not sure whether if I added `data-init-function="bpFieldInitxxxxElement"` it will working as expected or not, so I'm just using `$("#select_template")` , just like in this line above:

https://github.com/Laravel-Backpack/PageManager/blob/1610729c637a82c2d757ea347b8618ca9fd3e48c/src/resources/views/vendor/backpack/crud/fields/select_page_template.blade.php#L47

Reply:
@mahasadhu you are correct, the field isn't ready for being used multiple times, you did good by changing just the required to fix the issue 👌
@tabacitu I'll add an issue for 4.2 to make this field usable multiple times. For now we can merge this fix and let future us deal with the other problem 😅
Let me know if you agree with that.

Reply:
Issue added; https://github.com/Laravel-Backpack/CRUD/issues/3461
Just reviews the PR and it works as expected.

Reply:
Totally agree - hotfix now, let future us will refactor it to work multiple times per page  😉 Thanks a lot guys!